Q/QuantumEspresso/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+using BinaryBuilder, Pkg
+
+name = "QuantumEspresso"
+version = v"6.7.0"
+
+sources = [
+   ArchiveSource("https://github.com/QEF/q-e/releases/download/qe-6.7.0/qe-6.7-ReleasePack.tgz",
+                 "8f06ea31ae52ad54e900a2f51afd5c70f78096d9dcf39c86c2b17dccb1ec9c87")
+]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+    cd qe-*
+
+    export FFTW_INCLUDE=$prefix/include FFT_LIBS="-L$prefix/lib -lfftw3"
+    export BLAS_LIBS="-L$prefix/lib -lopenblas"
+    export LAPACK_LIBS="-L$prefix/lib -lopenblas"
```

Comment:
Please always use the most specific environment variables
```suggestion
    export FFTW_INCLUDE=${includedir} FFT_LIBS="-L${libdir} -lfftw3"
    export BLAS_LIBS="-L${libdir} -lopenblas"
    export LAPACK_LIBS="-L${libdir} -lopenblas"
```